inc/namespace.php

```diff
@@ -31,7 +31,10 @@
  */
 function bootstrap() {
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_elasticpress', 4 );
-	add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
+	add_action( 'plugins_loaded', function () {
+		// Run healthchecks on indexes after ElasticPress is loaded.
+		add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
```

Comment:
Could the filter be moved into the `load_elasticpress` function or is that too early (because the priority is < 10).

Reply:
I could test it again, it didn't seem to work when I tried it though, I think ElasticPress does some set up on `plugins_loaded` too though I didn't confirm that.